templates/repo/commits_list.tmpl

```diff
@@ -4,8 +4,9 @@
 				<tr>
 					<th class="three wide">{{ctx.Locale.Tr "repo.commits.author"}}</th>
 					<th class="two wide sha">SHA1</th>
-					<th class="nine wide message">{{ctx.Locale.Tr "repo.commits.message"}}</th>
+					<th class="eight wide message">{{ctx.Locale.Tr "repo.commits.message"}}</th>
 					<th class="two wide right aligned">{{ctx.Locale.Tr "repo.commits.date"}}</th>
+					<th class="one wide right aligned">{{ctx.Locale.Tr "repo.commit.operations"}}</th>
```

Comment:
Sounds a bit weird to waste so much space simply because of the column title (that isn't even particularly helpful).
What about leaving it empty instead?
```suggestion
					<th class="one wide right aligned"></th>
```

Reply:
I was thinking about getting rid of the whole table header. For example, branch list is without header as well.

Reply:
Is this tiny PR just being blown up to a full commit list redesign? 😆

Reply:
I think the first action is simply not including this column header.
The rest can be left for other PRs.

Reply:
I guess it looks alright even without the header here.

Reply:
I think in the future we should get rid of the HTML table completely. For now - I'm fine with leaving the title empty.

Reply:
Yes, looks acceptable:

<img width="188" alt="Screenshot 2023-09-29 at 21 52 42" src="https://github.com/go-gitea/gitea/assets/115237/f3abbd49-bb24-4cc1-8d05-6e01a6a1adf2">